### PR TITLE
Adding V.4.0.0

### PR DESCRIPTION
**Pull request by imohamme**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by imohamme** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,8 +9,8 @@ package:
 
 source:
 
-  url: https://github.com/nasa/NASAaccess/archive/V3.4.3.tar.gz
-  sha256: 56a4c30935d08aad56c28fd23874ae9018528179eccd82528a305a1bf7ebbfe3
+  url: https://github.com/nasa/NASAaccess/archive/v4.0.0.tar.gz
```
Comment (mfansler) [inline recipe/meta.yaml]: Would make sense to use the `version` variable:

```suggestion
  url: https://github.com/nasa/NASAaccess/archive/v{{ version }}.tar.gz
```

**Comment by imohamme** [comment]
@conda-forge-admin, please rerender

**Review by mfansler — APPROVED** [review]

**Comment by imohamme** [comment]
@mfansler @dbast  
can you please merge V.4.0.0 to conda-forge. 
Thanks!

**Pull request merged**